### DISCUSSION TITLE
How to hide Live Preview and Visit URL for taxonomy tag?

### DISCUSSION THREAD

**Discussion by lotarbo** [statamic/cms#8373] [Q&A]
Hello, i need to hide this buttons in cp only for taxonomy tag, because i don't need it)
But i can't understand hot to do it?

**Comment by duncanmcclean** [comment]
Just checked the code and I'm actually not sure if you can for taxonomies. 

With entries, you can disable them showing by removing the URL structure in the collection's settings but with taxonomies, you can't configure the URL structure that is used, so it can't be used to disable the buttons.
